zproject/default_settings.py

```diff
@@ -123,6 +123,7 @@
 # File uploads and avatars
 DEFAULT_AVATAR_URI = "/static/images/default-avatar.png"
 DEFAULT_LOGO_URI = "/static/images/logo/zulip-org-logo.svg"
+DEFAULT_TWITTER_LOGO_URI = "/static/images/logo/twitter-logo.svg"
```

Comment:
The `static/images/logo` directory is meant for just the Zulip logo.  I'm not sure where something like this should go -- my understanding is would be a file that would break Zulip if it were deleted.

Just to make sure I understand, I assume this is meant to be a copy of the Twitter logo to use for tweets without a user avatar or something?  I'm pretty reluctant to add something of that form, because it's end up being a file we can never move.

@andersk ideas on how to manage this sort of thing?

Reply:
Oh, I see, no, this is used next to the timestamp to just indicate something is a tweet?

Could we use https://fontawesome.com/v4.7.0/icon/twitter?